java/security/audit/weak-ssl-context.yaml

```diff
@@ -0,0 +1,14 @@
+rules:
+- id: weak-ssl-context
+  metadata:
+    cwe: 'CWE-326: Inadequate Encryption Strength'
+    source_rule_url: https://find-sec-bugs.github.io/bugs.htm#SSL_CONTEXT
+  message: |
+    An insecure SSL context was detected. TLS versions 1.0, 1.1, and all SSL versions
+    are considered weak encryption and are deprecated.
+    Use SSLContext.getInstance("TLSv1.2") for the best security.
+  severity: WARNING
+  languages: [java]
+  patterns:
+  - pattern-not: SSLContext.getInstance("TLS1.2")
```

Comment:
what happens on 1.3?

Reply:
Oh, yeah 🤔  Good thought. Guess I should use `where-python`